### ISSUE TITLE
Resident Request Application Form - Mandatory Fields

### ISSUE THREAD

**Issue by dribblecastle** [FriendToFriend/Revision_1.0.0#19]
See attached image.   All highlighted fields are required.  

![ftfa resident request form - mandatory fields](https://cloud.githubusercontent.com/assets/6880310/6771650/81bd00d6-d0a1-11e4-9d67-14ec6071d374.png)

**Comment by timcuster** [comment]
Assuming we will enforce all required fields on the front end.

**Comment by alphad05** [comment]
Yea, that's the way i was thinking about it. Asked Brent to comfirm with Lisa but i kinda forgot what the response was.

```
 On Tuesday, March 24, 2015 2:05 PM, timcuster <notifications@github.com> wrote:
```

 Assuming we will enforce all required fields on the front end.—

**Comment by dribblecastle** [comment]
I agree as well.    I didn’t pass that by Lisa as its a technical detail she probably wouldn’t care about.     All she cares is that certain information is always captured. .
